docs/tools/offline/cli-test-debug.md

```diff
@@ -91,6 +124,17 @@ mbedgt: available tests for built 'K64F-ARM', location '.\build\tests\K64F\ARM'
         test 'TESTS-functional-test3'
 ```
 
+In case of icetea:
+
+```
+$ mbed test -m K64F -t GCC_ARM --icetea --run-list
+Available icetea tests for build 'K64F-GCC_ARM', location 'TEST_APPS'
+    test 'UDPSOCKET_BIND_PORT'
+    test 'TCPSOCKET_BIND_PORT'
+    test 'TCPSERVER_ACCEPT'
+    test 'TCPSOCKET_ECHOTEST_BURST_SHORT'
+```
+
 ### Compiling and running tests
 
 You can specify to only **build** the tests by using the `--compile` option:
```

Comment:
"by using" -> with

This is nitpicky, admittedly. "By using" isn't wrong, just a little wordy.